arlo_server/routes.py

```diff
@@ -1053,8 +1052,9 @@ def audit_reset(election_id):
 @app.route("/auditboard/<passphrase>", methods=["GET"])
 def auditboard_passphrase(passphrase):
     auditboard = AuditBoard.query.filter_by(passphrase=passphrase).one()
+    set_loggedin_user(UserType.AUDIT_BOARD, auditboard.passphrase)
```

Comment:
Although the passphrase is unique right now, I would key the user on `auditboard.id` instead of `passphrase`. I don't see any advantage to keying it on passphrase.

In case it's useful, pointing out that the cookie is signed, so there's no way to forge the `auditboard.id`. So there is no authentication advantage to using `passphrase` in the cookie.

Reply:
Yep, that makes sense. Really not sure why I chose `passphrase` over `id` in the first place